src/node_file.cc

```diff
@@ -2085,7 +2088,8 @@ static void ReadBuffers(const FunctionCallbackInfo<Value>& args) {
 
   // Init uv buffers from ArrayBufferViews
   for (uint32_t i = 0; i < iovs.length(); i++) {
-    Local<Value> buffer = buffers->Get(env->context(), i).ToLocalChecked();
+    Local<Value> buffer =
+        buffers->Get(env->context(), i).FromMaybe(Local<Value>());
```

Comment:
This is trading a potential assertion failure for a potential segfault – that’s not good. I’d do proper error handling here, i.e. return if there’s an exception, or leave it as-is. (Same in the chunk above.)